apps/api/schedule.py

```diff
@@ -186,3 +237,4 @@ def get(self):
 api.add_resource(FavouriteExternal, "/external/<int:event_id>/favourite")
 api.add_resource(ScheduleMessage, "/schedule_messages")
 api.add_resource(UpdateLotteryPreferences, "/schedule/tickets/<proposal_type>/preferences")
+api.add_resource(C3VOCPublishingWebhook, "/c3voc/publishing-webhook")
```

Comment:
To align with some existing webhook URLs (for example [this Stripe webhook endpoint](
https://github.com/emfcamp/Website/blob/888fb8b7d63c7560376bc2d5670b0e46f61fdce0/apps/payments/stripe.py#L171-L172)), and to indicate the key affected resource in the path, I'd suggest:

```suggestion
api.add_resource(ProposalC3VOCWebhookResource, "/proposal/c3voc-webhook")
```